### DISCUSSION TITLE
在pcl内群中重新加入.jrrp

### DISCUSSION THREAD

**Discussion by PSR2144-3933** [Meloong-Git/PCL#805] [提问] [answered]
龙猫你还我.jrrp!

**Comment by WForst-Breeze** [comment, marked as answer]
可能不会添加哦，以前群友聊着聊着天就被 .jrrp 和 .upd 打断了（）导致群里聊天体验很不好
这或许也是龙猫把更新密钥获取（.upd）移动到了爱发电的原因吧🧐

**Comment by Light-Beacon** [comment]
兔兔暴毙风险太大，实在没办法

**Comment by wuliaodexiaoluo** [comment]
QQ 在切版本，风控很严重。。。

**Comment by Ad-closeNN** [comment]
> QQ 在切版本，风控很严重。。。

你又在挖坟了

btw:
.jrrp 和 .upd是啥 上一年10月才进的内群

**Reply by MoYuan-CN** [reply]
.jrrp -> 今日人品
.upd -> 拿更新密钥的

**Reply by wuliaodexiaoluo** [reply]
~挖坟使我快乐~
